lib/vmdb/loggers.rb

```diff
@@ -73,6 +81,7 @@ def self.create_loggers
     def self.create_multicast_logger(log_file_path, logger_class = VMDBLogger)
       logger_class.new(log_file_path).tap do |logger|
         logger.extend(ActiveSupport::Logger.broadcast($container_log)) if ENV["CONTAINER"]
+        logger.extend(ActiveSupport::Logger.broadcast($journald_log))  if MiqEnvironment::Command.supports_systemd?
```

Comment:
Is this simpler if we just do `if $journald_log`?  Then we can eliminate proliferation of the conditional.

Reply:
:+1: done